### PR TITLE
Allow to work without activesupport concerns

### PR DESCRIPTION
**Pull request by rmulligan**

### CODE REVIEW

**Comment by dblock** [comment]
This broke the build, please check.

**Comment by rmulligan** [comment]
Need to refactor tests. Will fix. Thanks.

**Comment by dblock** [comment]
This is green, nice. So can we remove https://github.com/dblock/slack-ruby-bot/blob/master/slack-ruby-bot.gemspec#L19? Also needs a CHANGELOG entry.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 * [#54](https://github.com/dblock/slack-ruby-bot/pull/54): Improvements to Hook configuration - [@dramalho](https://github.com/dramalho).
 * Your contribution here.
 
+### 0.7.1 (7/7/2016)
```
Comment (dblock) [inline CHANGELOG.md]: Don't call a release, you're not making a release. This should be under 0.8.1 (Next).

Comment (rmulligan) [inline CHANGELOG.md]: Thanks. Sorry, my first contribution to open source.

Comment (dblock) [inline CHANGELOG.md]: You're doing totally fine!

Comment (dblock) [inline CHANGELOG.md]: And much thanks.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,8 @@
 * [#75](https://github.com/dblock/slack-ruby-bot/issues/75): Guarantee order of command evaluation - [@dblock](https://github.com/dblock).
 * [#76](https://github.com/dblock/slack-ruby-bot/issues/76): Infinity error when app disabled - [@slayershadow](https://github.com/SlayerShadow).
 * [#81](https://github.com/dblock/slack-ruby-bot/pull/81): Remove bundler dependency - [@derekddecker](https://github.com/derekddecker).
-* Your contribution here.
```
Comment (dblock) [inline CHANGELOG.md]: Put this back please for the next person.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,7 +4,8 @@
...
+* [#84](https://github.com/dblock/slack-ruby-bot/pull/84): Removed dependency on ActiveSupport. - [@rmulligan](https://github.com/rmulligan).
```
Comment (dblock) [inline CHANGELOG.md]: Remove the period after ActiveSpport. Sorry for nitpicking, make it look the same as everything else.

**Comment by dblock** [comment]
Merged.

**Comment by dblock** [comment]
This broke logging, see https://github.com/dblock/slack-ruby-bot/issues/85.

**Pull request merged**